app/views/content/life-as-a-teacher/_categories.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+  <section class="category col-space-m">
+    <section class="container category__cards main-section">
+      <nav class="category__nav-cards">
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h3", card:
```

Comment:
This page goes straight from an h1 to an h3 @spencerldixon which is not good in terms of accessibility

The nav card has to be an h2 to avoid skipped heading levels?

![image](https://github.com/user-attachments/assets/028c1022-8ea4-4b83-b607-4c60d10b8c23)